CHANGELOG.md

```diff
@@ -1,6 +1,47 @@
 # TiSpark Changelog
 All notable changes to this project will be documented in this file.
 
+## [TiSpark 3.0.0] 2022-06-15
+### Compatibility Changes
+- TiSpark without catalog plugin is no more supported. You must configure catalog configs and use tidb_catalog now [#2252](https://github.com/pingcap/tispark/pull/2252)
+- TiSpark's jar has a new naming rule like `tispark-assembly-{$spark_version}_{$scala_version}-{$tispark_verison}` [#2370](https://github.com/pingcap/tispark/pull/2370)
+
+### New Feature
+- Support DELETE statement [#2276](https://github.com/pingcap/tispark/pull/2276)
+- Support Spark 3.2 [#2287](https://github.com/pingcap/tispark/pull/2287)
+- Support telemetry to collect information [#2316](https://github.com/pingcap/tispark/pull/2316)
+- Support stale read to read historical versions of data [#2322](https://github.com/pingcap/tispark/pull/2322)
+- Support TLS with reload capability [#2306](https://github.com/pingcap/tispark/pull/2306) [#2349](https://github.com/pingcap/tispark/pull/2349) [#2365](https://github.com/pingcap/tispark/pull/2349) [#2377](https://github.com/pingcap/tispark/pull/2377)
+
+### Fixes
+- Fix the wrong result of _tidb_rowid when set `spark.tispark.show_rowid=true` [#2270](https://github.com/pingcap/tispark/pull/2270)
+- Fix sum not push down bug [#2314](https://github.com/pingcap/tispark/pull/2314)
+- Fix limit not push down bug [#2329](https://github.com/pingcap/tispark/pull/2329)
```

Comment:
Is there any syntax error here? How about "Fix sum can't be pushed down bug"?

Reply:
Fix the bug that the sum filter can't be pushed down

Reply:
For consistency with releases and release emails. I suggest not change it now.